<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>chat</class>
 <widget class="QWidget" name="chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>353</width>
    <height>329</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>محادثه</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../pathIcon.qrc">
    <normaloff>:/images/chat.png</normaloff>:/images/chat.png</iconset>
  </property>
  <widget class="QTextBrowser" name="textBrowserShowMessage">
   <property name="geometry">
    <rect>
     <x>6</x>
     <y>10</y>
     <width>341</width>
     <height>271</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="textMessage">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>290</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="labConnectServer">
   <property name="geometry">
    <rect>
     <x>305</x>
     <y>290</y>
     <width>41</width>
     <height>31</height>
    </rect>
   </property>
   <property name="toolTip">
    <string>الاتصال بل سيرفير</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap resource="../pathIcon.qrc">:/images/connect.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="idUser">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>360</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>LabUserID</string>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="../pathIcon.qrc"/>
 </resources>
 <connections/>
</ui>
